### ISSUE TITLE
LeakCanary pointing on net.grandcentrix.thirtyinch.internal.PresenterSavior.INSTANCE

### ISSUE THREAD

**Issue by ar-g** [grandcentrix/ThirtyInch#68]
We got several cases when LeakCanary pointing on `net.grandcentrix.thirtyinch.internal.PresenterSavior.INSTANCE`. Do you have any suggestions where we should look in?

```
* features.profile.my.sold.MySoldAdsFragment has leaked:
* GC ROOT static net.grandcentrix.thirtyinch.internal.PresenterSavior.INSTANCE
* references net.grandcentrix.thirtyinch.internal.PresenterSavior.mPresenters
* references java.util.HashMap.table
* references array java.util.HashMap$HashMapEntry[].[13]
* references java.util.HashMap$HashMapEntry.value
* references features.profile.my.favorites.MyFavoritesAdsPresenter.mView
* references features.profile.my.favorites.MyFavoritesAdsFragment.mFragmentManager
* references android.support.v4.app.FragmentManagerImpl.mActive
* references java.util.ArrayList.elementData
* references array java.lang.Object[].[2]
* leaks features.profile.my.sold.MySoldAdsFragment instance
```

```
* features.chats.list.ChatListFragment has leaked:
* GC ROOT static net.grandcentrix.thirtyinch.internal.PresenterSavior.INSTANCE
* references net.grandcentrix.thirtyinch.internal.PresenterSavior.mPresenters
* references java.util.HashMap.table
* references array java.util.HashMap$HashMapEntry[].[7]
* references java.util.HashMap$HashMapEntry.value
* references features.chats.list.ChatListPresenter.mView
* leaks features.chats.list.ChatListFragment instance
```

**Comment by passsy** [comment]
duplicate, see #33. it happens when you detach the fragment

**Comment by passsy** [comment]
fixed in [v0.8.0-rc4](https://github.com/grandcentrix/ThirtyInch/releases/tag/v0.8.0-rc4)
